packages/host/tests/integration/components/input-field-test.gts

```diff
@@ -0,0 +1,308 @@
+import { module, test } from 'qunit';
```

Comment:
Here I add a new set of test that checks input validation of BigInt. These test should be generalised when adding the resource